Client/WarFare/GameBase.h

```diff
@@ -21,6 +21,7 @@ class CGameBase : public CN3Base
 	static CN3TableBase<__TABLE_FX>					s_pTbl_FXSource;		// FX소스 정보 테이블..
 	static CN3TableBase<__TABLE_QUEST_MENU>			s_pTbl_QuestMenu;		// 퀘스트 선택 메뉴
 	static CN3TableBase<__TABLE_QUEST_TALK>			s_pTbl_QuestTalk;		// 퀘스트 지문
+	static CN3TableBase<__TABLE_QUEST_CONTENT>		s_pTbl_Quest_Content;	// Quest Content Table
```

Comment:
I think all 3 of these names should be named consistently. They should either be Quest_Menu, Quest_Talk, Quest_Content etc, or QuestMenu, QuestTalk, QuestContent.
Just because they're already named like this, I would suggest just renaming this to s_pTbl_QuestContent for consistency.